src/algorithms/graph-builder.ts

```diff
@@ -0,0 +1,37 @@
+import {INode, Node} from '../api/schema/node';
+import {IWay, Way} from '../api/schema/way';
+
+function randNode(id: number) {

```

Comment:
```suggestion
function randNode(id: number): Node {
```
+1 for using typed input parameters but it would be nice if you also specify the type of the return value